tests/basic/pure_test.go

```diff
@@ -5130,3 +5130,125 @@ var _ = Describe("{ValidatePodNameinVolume}", func() {
 		AfterEachTest(contexts)
 	})
 })
+var _ = Describe("{MultiTenancyFATestWithPodRealm}", func() {
```

Comment:
we can rename this more appropriately

```suggestion
var _ = Describe("{FAMultiTenancyMultiAppWithPodRealm}", func() {
```